test/library/packages/TensorFlow/TensorFlow.chpl

```diff
@@ -0,0 +1,776 @@
+module TensorFlow {
+  module C_TensorFlow {
+    require  "tensorflow/c/c_api.h", "-ltensorflow";
```

Comment:
What's the plan for making tensorflow available on the testing machines?

Do we need some kind of `skipif` if tensorflow is not available?

Reply:
Ah, I missed that this is notested.

Reply:
It does need a `skipif` and I forgot to add one.  The `.notest` is just for `Tensorflow.chpl`, not the `load_graph.chpl` test that uses it.  The usual one that we used for other Library tests isn't working for me though for some reason.  `DYLD_LIBRARY_PATH` doesn't seem to be set from within python even though I do have it set.

Reply:
Could you check the result of `python -m tensorflow` within the skipif? I still think we should track that this test isn't running via an issue until we set up a testing config with tensorflow available.

Reply:
This is with DYLD_LIBRARY_PATH set to point at the location where libtensorflow.so is installed.  I think a more standard install would find it.

```bash
% python -m tensorflow
/Users/diten/anaconda2/bin/python: No module named tensorflow
```

I opened #11797 to request installing TensorFlow on a test system.